transactions/transfer_many_accounts.cdc

```diff
@@ -0,0 +1,36 @@
+import FungibleToken from 0xFUNGIBLETOKENADDRESS
+import ExampleToken from 0xTOKENADDRESS
+
+transaction(amount: UFix64, to: [Address]) {
```

Comment:
In  [the minimum account "migration"](https://github.com/dapperlabs/flow-internal/issues/1354) we will be depositing a different amount to each account, so the input of the method could actually be: `addressAmountMap: {Address: UFix64}`. the same goes for the private transfer.